test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs

```diff
@@ -84,7 +84,11 @@ public void TestBuild()
                     "Test invalid autolink style xref with anchor: <xref:invalid#anchor>",
                     "Test short xref: @XRef2",
                     "Test xref with query string: <xref href=\"XRef2?text=Foo%3CT%3E\"/>",
+                    "Test invalid xref with query string: <xref href=\"invalid?alt=Foo%3CT%3E\"/>",
                     "Test xref with attribute: <xref href=\"XRef2\" text=\"Foo&lt;T&gt;\"/>",
+                    "Test xref with attribute: <xref href=\"XRef2\" name=\"Foo&lt;T&gt;\"/>",
+                    "Test invalid xref with attribute: <xref href=\"invalid\" alt=\"Foo&lt;T&gt;\"/>",
+                    "Test invalid xref with attribute: <xref href=\"invalid\" fullname=\"Foo&lt;T&gt;\"/>",
```

Comment:
html attribute is case insensitive, usually use all lower case.

---
In reply to: [84835709](https://github.com/dotnet/docfx/pull/823#discussion_r84835709) [](ancestors = 84835709)